JenkinsConsoleUtility/jcuSrc/Commands/VersionVarWriter.cs

```diff
@@ -41,16 +42,16 @@ public int Execute(Dictionary<string, string> argsLc, Dictionary<string, string>
                 _apiSpecPfUrl = JenkinsConsoleUtility.GetArgVar(argsLc, "apiSpecPfUrl");
             else
             {
-                JenkinsConsoleUtility.FancyWriteToConsole("Api-Spec input not defined.  Please input one of: apiSpecPath, apiSpecGitUrl, apiSpecPfUrl");
+                JcuUtil.FancyWriteToConsole("Api-Spec input not defined.  Please input one of: apiSpecPath, apiSpecGitUrl, apiSpecPfUrl");
```

Comment:
nit: I think I have seen JcuUtil in a few other places, but wouldn't it properly shorten to JcUtil?

Reply:
Lol
You don't like Jenkins Console Utility Util?